### ISSUE TITLE
vagrant 'quick start' not working

### ISSUE THREAD

**Issue by tommyk** [openwhisk/openwhisk#956]
## Environment details:

* local checkout from master on OSX
* vagrant version: Vagrant 1.8.1

## Steps to reproduce the issue:

1.   Run './hello' or 'vagrant up' from 'tools/vagrant' folder from MASTER

## Provide the expected results and outputs:

```
waiting for

wsk action invoke /whisk.system/samples/echo -p message hello --blocking --result
{
    "message": "hello"
}

from what the docs say
```

## Provide the actual results and outputs:

```
Getting "The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong."  I see the following when in the console 
'GPG error: https://apt.dockerproject.org ubuntu-trusty InRelease: The following signatures 
couldn't be verified because the public key is not available: NO_PUBKEY F76221572C52609D'.  
```

Not sure if that is the issue, any ideas? Just trying to run a hello world...

## Additional information you deem important:
* first time running this, just seemed like something was up since the first command does not even run... 

Any ideas?

**Comment by tommyk** [comment]
After looking more, the problem lies in the gradle.sh file in 'tools/ubuntu-setup'.  Here is the error "Unable to locate package gradle-2.7".  Going to keep poking and updating...

**Comment by rabbah** [comment]
Try changing gradle.sh:

```
diff --git a/tools/ubuntu-setup/gradle.sh b/tools/ubuntu-setup/gradle.sh
index 019efae..e3e9e18 100755
--- a/tools/ubuntu-setup/gradle.sh
+++ b/tools/ubuntu-setup/gradle.sh
@@ -3,5 +3,5 @@ sudo apt-get install -y python-software-properties
 sudo add-apt-repository -y ppa:cwchien/gradle 
 sudo apt-get update -y
 sudo apt-cache search gradle 
-sudo apt-get install -y gradle-2.7 
+sudo apt-get install -y gradle-2.11
```

**Comment by tommyk** [comment]
Yea, i bumped to 2.10 and it worked.  Should I put a patch in to update the version, or is this going to make you guys go another route for installing?

**Comment by markusthoemmes** [comment]
@rabbah i noted in a different issue that gradle doesn't need to be installed at all as you mentioned.

**Comment by rabbah** [comment]
@tommyk looks like we will just remove this file to avoid the hiccup in the future.

**Comment by csantanapr** [comment]
I agree we should tell everyone to use gradlew, and the wrapper will take of downloading and installing the correct version of gradle.

**Comment by rabbah** [comment]
@csantanapr proposed patch: https://github.com/openwhisk/openwhisk/compare/master...rabbah:gradlew?expand=1

we should use this issue to drive the vagrant ci (which would have caught the bit rot).

**Comment by csantanapr** [comment]
hehe I was already looking into it and I have the same patch 👍 
But in addition all the places (i.e. ansible/README) we doc "gradle" replace with "./gradlew"
Was also looking at what version of gradle we currently use (so far it's 2.12)
testing now...
 
Yes we need CI

**Comment by csantanapr** [comment]
Hi @tommyk this should be fixed now as of a few hours with this commit 86551b61c6056b07faf1d3a5f4126fa86a9057b2

thanks for your patient

**Comment by tommyk** [comment]
Awesome, thanks guys!
